package.json

```diff
@@ -346,7 +346,8 @@
         "visible": true,
         "subPermissions": [
           "ui-orders.settings.order-templates.edit",
-          "orders.order-templates.item.post"
+          "orders.order-templates.item.post",
+          "templates.item.post"
```

Comment:
Why should the user with this perm set be able to interact with the `template-engine` interface? AFAIR order templates don't call any endpoint from that interface.

Reply:
You must have wanted to assign this permission to the set that is used to interact with the list configuration.

Reply:
We use templates [/list-configuration](https://folio-snapshot.dev.folio.org/settings/orders/list-configuration)

Reply:
how is this permission connected to order templates?

Reply:
I would suggest to remove this permission and open ticket for permissions, ui-orders.settings.view is not right one to control edit of these settings

Reply:
@NikitaSedyx Let me sync with the BE team about the endpoint.

Reply:
@alisher-epam 
focus on the last comment